docs/source/reference/pdk_configuration.md

```diff
@@ -36,37 +63,52 @@ Any examples provided are for the `sky130A` PDK.
-| `FP_PDN_CORE_RING_VSPACING` | Defines the spacing for the vertical layer used to create the core ring in the PDN. <br> (Example: `5`) | |
-| `FP_PDN_CORE_RING_HSPACING` | Defines the spacing for the horizontal layer used to create the core ring in the PDN. <br> (Example: `5`) | |
-| `FP_PDN_CORE_RING_VOFFSET` | Defines the offset for the vertical layer used to create the core ring in the PDN. <br> (Example: `20`) | |
-| `FP_PDN_CORE_RING_HOFFSET` | Defines the offset for the horizontal layer used to create the core ring in the PDN. <br> (Example: `20`) | |
-| `WIRE_RC_LAYER` | The metal layer used in estimate parastics `set_wire_rc`. <br> (Example: `met1`) ||
-| `GRT_LAYER_ADJUSTMENTS` | Layer-specific reductions in the routing capacity of the edges between the cells in the global routing graph, delimited by commas. Values range from 0 to 1. <br> (Example: `0.99,0,0,0,0,0`)
+| `WIRE_RC_LAYER` | The metal layer used in estimate parastics `set_wire_rc`. <br> (Example: `met1`) |
 | `FP_IO_HLAYER`  | The metal layer on which to place the io pins horizontally (top and bottom of the die). <br>(Example: `met3`)|
 | `FP_IO_VLAYER`  | The metal layer on which to place the io pins vertically (sides of the die) <br> (Example: `met2`)|
 | `FP_TAPCELL_DIST`  | The horizontal distance between two tapcell columns <br> (Default: `14`) |
-| `RT_MIN_LAYER`  | The lowest metal layer to route on. <br>(Example: `met1`)|
-| `RT_MAX_LAYER`  | The highest metal layer to route on. <br> (Example: `met5`)|
 | `RCX_RULES_MIN` | OpenRCX rules at the minimum corner. (Optional) |
 | `RCX_RULES` | OpenRCX rules at the nominal corner. |
 | `RCX_RULES_MAX` | OpenRCX rules at the maximum corner. (Optional) |
-| `WIRE_LENGTH_THRESHOLD` | A value in microns above which wire lengths generate warnings, and, if `QUIT_ON_LONG_WIRE` is set, the flow will error out. If a PDK does not set this value, the value is considered to be infinite. (Optional) |
 
 ## SCL-specific variables
 
-This section defines the necessary variables to configure a standard cell library for use with OpenLane:
+This section defines the necessary variables to configure a standard cell library for use with OpenLane.
+
+### User-modifiable
+
+These values may be modified by the user configuration and the
+included values should be considered "defaults."
+
+| Variable      | Description                                                   |
+|---------------|---------------------------------------------------------------|
+| `MAX_TRANSITION_CONSTRAINT` | Defines the maximum slew (transition) value in ns. |
+| `MAX_FANOUT_CONSTRAINT` | Defines the maximum fanout for a single output in the design. |
+| `OUTPUT_CAP_LOAD` | Defines the capacitive load on the output ports in fF. |
+| `CTS_MAX_CAP` | Defines the maximum capacitance for clock tree synthesis in the design in fF. |
```

Comment:
pF according to https://github.com/The-OpenROAD-Project/OpenLane/blob/2b355aaab67eebb8c74b00258b775c710b07b861/scripts/openroad/cts.tcl#L33